src/main/java/com/cityelf/model/water/parser/Place.java

```diff
@@ -0,0 +1,23 @@
+package com.cityelf.model.water.parser;
```

Comment:
Since it is not database related class, let's move it to a domain package (create another folder domain in the com/cityelf)